docs/UserGuide.md

```diff
@@ -287,17 +287,15 @@ Format: `sort by/SORT_KEY in/SORT_ORDER`
   - `priority` - sort by the priority of the task
   - `name` - sort by the name of the task
 - `SORT_ORDER` is the order in which the tasks are listed out.
-  - `ascending`
-  - `descending`
+  - `asc` - sort in ascending order
+  - `desc` - sort in descending order
 
 Example:
-- `sort by/deadline in/ascending`: lists the tasks with the earliest deadline first
-- `sort by/priority in/descending`: lists the tasks with the highest priority first
+- `sort by/deadline in/asc`: lists the tasks with the earliest deadline first
+- `sort by/priority in/desc`: lists the tasks with the highest priority first
 
 <div markdown="span" class="alert alert-primary">:bulb: **Note:**
-You can use the following abbreviations to sort tasks faster.
-- `asc` in place of `ascending`
-- `desc` in place of `descending`
+The tasks will preserve the specified sorting until a different `sort` command is give.
```

Comment:
Should this be `...until a different sort command is given`?

Reply:
Yep, you are right! Will change!